src/com/google/enterprise/adaptor/filenet/FileNetAdaptor.java

```diff
@@ -28,6 +28,7 @@
 import com.google.enterprise.adaptor.DocId;
 import com.google.enterprise.adaptor.DocIdPusher;
 import com.google.enterprise.adaptor.InvalidConfigurationException;
+import com.google.enterprise.adaptor.Principal;
```

Comment:
I would definitely statically import DEFAULT_NAMESPACE rather than not import the java.security.Principal class when both are used. If we need more from Principal then we can deal with it, but we probably don't (no v4 connector currently does, AFAIK).